src/modules/submissions.js

```diff
@@ -1135,12 +1135,24 @@
 			} );
 		}
 
+		function checkForBlocks() {
+			return afchSubmission.getSubmitter().then( function ( creator ) {
+				return checkIfUserIsBlocked( creator ).then( function ( blockData ) {
+					if ( blockData !== null ) {
+						var warning = creator + ' was blocked by ' + blockData.by + ' with an expiry time of ' + blockData.expiry + '. Reason: ' + blockData.reason;
```

Comment:
<img width="1023" alt="Screenshot 2022-02-20 at 12 26 48 AM" src="https://user-images.githubusercontent.com/31818903/154814991-94ff8285-b599-4dfa-a3df-12e7f6f8083b.png">

blockData.expiry is in ISO format which looks ugly when exposed in UI. Could it be changed to a more readable format? 

Fine to leave for later though.